proposals/dao/fip_99.ts

```diff
@@ -0,0 +1,137 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { contract } from '@openzeppelin/test-environment';
+import { Fei } from '@custom-types/contracts';
+import { getImpersonatedSigner } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+/*
+
+DAO Proposal #99
+
+Description: Sell RAI to replenish DAI reserves and/or relieve upcoming peg pressure
+
+Steps:
+  1 - Move 80% of PCV RAI to AAVE from Fuse Pool 9 via the ratioPCVControllerV2
+  2 - Call depsoit() on the aave rai pcv deposit
+  3 - Grant the MINTER role to the rai psm
+  4 - Grant the PCV_CONTROLLER role to the rai pcv drip controller
+  5 - Pause mint on the rai psm
+  6 - Whitelist the fuse pool 9 rai pcv deposit on the pcv guardian
+  7 - Whitelist the aave rai pcv deposit on the pcv guardian
+*/
+
+const fipNumber = '99'; // Change me!
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const priceBoundPSMFactory = await ethers.getContractFactory('PriceBoundPSM');
+  const raiPriceBoundPSM = await priceBoundPSMFactory.deploy(
+    30_000,
+    ethers.constants.WeiPerEther,
+    {
+      coreAddress: addresses.core,
+      oracleAddress: addresses.chainlinkRaiUsdCompositeOracle,
+      backupOracle: ethers.constants.AddressZero,
+      decimalsNormalizer: 0,
+      doInvert: false
+    },
+    0, // mint fee = 0 bp
+    0, // redeem fee = 0 bp
+    ethers.constants.WeiPerEther.mul(10_000_000), // reserves threshold = 10m
+    50_000, // fei rate limit = 50k fei/sec
+    0, // minting buffer cap = 0 = disables minting fei
+    addresses.rai, // rai as the underlying token
+    addresses.feiDAOTimelock // surplus target = timelock
+  );
+  await raiPriceBoundPSM.deployed();
+
+  logging && console.log(`Rai price bound psm deployed`);
+
+  const pcvDripControllerFactory = await ethers.getContractFactory('PCVDripController');
+  const raiPCVDripController = await pcvDripControllerFactory.deploy(
+    addresses.core,
+    addresses.aaveRaiPCVDeposit, // drip source = aave rai pcv deposit
+    raiPriceBoundPSM.address, // drip target = rai psm
```

Comment:
The whole point of the NonCustodial PSM is to have 0 opportunity cost of capital. Why are we adding a PCVDripController here when the PSM will just pull from the PCV Deposit?

Reply:
(we're not using the non-custodial psm here)